zoo/gobigger/env/gobigger_rule_bot.py

```diff
@@ -0,0 +1,215 @@
+import copy
+from ding.policy.base_policy import Policy
+from ding.utils import POLICY_REGISTRY
+import torch
+import math
+import queue
+import random
+from pygame.math import Vector2
```

Comment:
can we don't add pygame requirement here?

Reply:
use numpy instead of pygame